.storybook/main.js

```diff
@@ -19,4 +19,20 @@
 module.exports = {
 	stories: [ '../stories/**/*.stories.js', '../assets/js/**/*.stories.js' ],
 	addons: [ '@storybook/addon-viewport', '@storybook/addon-postcss' ],
+	previewHead: ( head ) => {
+		if ( ! process.env.VRT ) {
```

Comment:
Not strictly necessary, but this might be a bit more user-friendly as something like:

```suggestion
		if ( ! process.env.VRT?.length && process.env.VRT !== '0' ) {
```

Otherwise won't `VRT= npm run build:storybook` or `VRT=0 npm run build:storybook` cause this to be truthy? That would be a bit unexpected—even though we don't use it now it might be nice to anticipate that usage and handle it here 🙂